test/test_quantized.py

```diff
@@ -147,25 +147,30 @@ def test_qadd_scalar_relu(self, A, b):
         add_scalar_relu = torch.ops.quantized.add_scalar_relu
 
         A, (scale, zero_point, dtype) = A
+        check_val = zero_point + b
+        if dtype == torch.qint8 and check_val < -128 or check_val > 127:
+            return
+        if dtype == torch.quint8 and check_val > 255 or check_val < 0:
+            return
+        if dtype == torch.quint8 and check_val > 2**31 - 1 or check_val < -(2**31):
```

Comment:
L 155: We should not support scalar_add for qint32.  Currently, qint32 support is restricted to zero-point = 0.